### ISSUE TITLE
Allow x!backup load to select from the last 5/10 backups

### ISSUE THREAD

**Issue by ThatLifeofGames** [Xenon-Bot/xenon-main#26]
When running x!backup load on it's own, you could create a "menu" of the last 5 or 10 backups, with reactions to select the ID to load.

I made a little image to show roughly what I mean. Obviously could be much fancier but this is lazy inspect element. _I also couldn't be arsed to find numbers so imagine the letters are numbers_.
![image](https://user-images.githubusercontent.com/35865509/93332809-4fce9c00-f81a-11ea-824b-4d609997405c.png)

**Comment by ThatLifeofGames** [comment]
The reactions could also be replaced with a proper UI menu which discord teased a while ago.

**Comment by Merlintor** [comment]
Yeah probably not worth implementing before discord releases their new UI interaction stuff. Nice idea tho
